pyntcloud/io/stl.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+from stl import mesh
+import pandas as pd
+
+def read_stl(filename):
+    stl = mesh.Mesh.from_file(filename)
+    length1 = np.size(stl.x.flatten(),0)
+    #length2 = np.size(stl.normals.flatten(),0)
+#    length3 = np.size(stl.v0.flatten())
+#    
+    index1 = pd.RangeIndex(start=0,stop=length1)
+    #index3 = pd.RangeIndex(start=0,stop=length3)
+
+    x = stl.x.flatten()
```

Comment:
I've taken a look at numpy-stl documentation and I think that this does not get the correct result.

A point cloud is made of **unique** points but in the `Mesh` object created by `numpy-stl` this information is somehow hidden. Probably because the library is focused on managing the faces rather than the vertices.

The cleanest way to extract the unique points from the Mesh object I've found is this:

```python
reduntant_points = np.vstack((mesh.v0, mesh.v1, mesh.v2))
points = np.vstack({tuple(row) for row in a})
# ...
# ...
data["points"] = pd.DataFrame(points, columns=["x", "y", "z"])
```